SnailWay/SnailWay.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SnailWay
+{
+    public class SnailWay
+    {
+        public static int[] CalculateWay(int[][] grid, List<int> snailWay)
+        {
+            snailWay.AddRange(grid[0]);
```

Comment:
`snailWay.Add()` can be used

Reply:
Yes but how should I add several numbers?